go-controller/pkg/ovn/external_ids_syncer/acl/acl_sync.go

```diff
@@ -304,7 +304,7 @@ func (syncer *aclSyncer) updateStaleNetpolNodeACLs(legacyACLs []*nbdb.ACL, exist
 
 func (syncer *aclSyncer) getNetpolGressACLDbIDs(policyNamespace, policyName, policyType string,
 	gressIdx, portPolicyIdx, ipBlockIdx int) *libovsdbops.DbObjectIDs {
-	return libovsdbops.NewDbObjectIDs(libovsdbops.ACLNetworkPolicy, syncer.controllerName,
+	objIDs := libovsdbops.NewDbObjectIDs(libovsdbops.ACLNetworkPolicy, syncer.controllerName,
```

Comment:
Not sure if that was intended, but this change came back

Reply:
@npinaeva yes, it is intended, because object id type `ACLNetworkPolicy` is changed, i.e. port-policy-id is removed, port-policy-protocol is added, if we don't change this part, test case will break, and I think acl syncer might break in runtime too. I added comment to explain.

Reply:
yeah so in that case you need a new dbIDs type. Similar to `AddressSetNetworkPolicy` which got deprecated and replaced with `AddressSetPodSelector`. Because syncer package is supposed to update no db index to the first version of db index, not to the latest version of db index.

You can rename existing `ACLNetworkPolicy` ids to something like `ACLNetworkPolicyPortIndex` and use it in sync package, and then create a new type `ACLNetworkPolicy` with the new ids

Reply:
sounds good, also added a new ownerType `NetworkPolicyPortIndexType`, otherwise the program will panic, hope it won't break anything. thanks!